spec/dummy/app/views/home/components.rb

```diff
@@ -24,7 +24,10 @@ def main
               li {
                 a(href: 'mailto:support@dobt.co') {
                   span 'Contact support&hellip;'.html_safe, class: 'drop_rt_item'
-                  span(class: 'drop_rt_arrow') { i(class: 'fa fa-external-link') }
+                  span(
+                    icon('external-link'),
```

Comment:
This isn't just decorative and carries semantic meaning, so we should have a label for it